src/App.vue

```diff
@@ -95,6 +96,30 @@ export default {
 	},
 
 	computed: {
+		// Computed properties for unread counts
```

Comment:
This comment doesn't have additional value. The code below is `computed`-s because it's literally the next line after `computed: {` and they are for `unread`-s as their names say.
```suggestion
```